jgclark.DailyJournal/src/journal.js

```diff
@@ -54,6 +56,29 @@ export async function dayStart(today: boolean = false): Promise<void> {
       return
     }
   }
+
+  // Get config settings from Template folder _configuration note
+  const journalConfig = await getOrMakeConfigurationSection(
+    'dailyJournal',
+    DEFAULT_JOURNAL_OPTIONS,
+    MINIMUM_JOURNAL_OPTIONS,
+  )
+  // console.log(JSON.stringify(journalConfig))
+  if (journalConfig == null
+    || Object.keys(journalConfig).length === 0) // this is how to check for empty object
+  {
+    console.log('\tWarning: Cannot find suitable \'dailyJournal\' settings in Templates/_configuration note. Stopping.')
+    await showMessage(
+      'Cannot find \'dailyJournal\' settings in _configuration.',
+      'Yes, I\'ll check my _configuration settings.',
+    )
+    return
+  }
+
+  pref_templateTitle = (journalConfig?.nameOfDailyNoteTemplate != null)
```

Comment:
Using two very different names for the same thing is confusing.
Can I suggest you use `templateTitle` not `nameOfDailyNoteTemplate` which better fits with the style of other settings, and matches the `pref_templateTitle` in the code?